qiita_pet/handlers/api_proxy/tests/test_sample_template.py

```diff
@@ -0,0 +1,161 @@
+from unittest import TestCase, main
+from os.path import join
+
+from qiita_core.qiita_settings import qiita_config
+from qiita_pet.handlers.api_proxy.sample_template import (
+    sample_template_get_req, sample_template_post_req,
+    sample_template_put_req, sample_template_delete_req,
+    sample_template_filepaths_get_req)
+
+
+class TestSampleAPI(TestCase):
+    def test_sample_template_get_req(self):
```

Comment:
Same here regarding status + message strategy, and similarly to all the other ones.